pact/verify_wrapper.py

```diff
@@ -165,9 +162,10 @@ def call_verify(
 
         if(provider_app_version):
             command.extend(["--provider-app-version",
-                            provider_app_version,
-                            "--publish-verification-results"])
-            # self.publish_results(provider_app_version, command)
+                            provider_app_version])
+
+        if kwargs.get('publish_verification_results', False):
```

Comment:
Do you need to explicitly check the value is True (as per the check for `verbose` on line 170)?. I must admit on first reading, I thought it was comparing to `False` which seemed backwards, but I think that's the default value.

Reply:
I think it works (per the unit tests) but I agree it's open to error and hard to read. I'll look at tweaking it.